src/makeInfinityAdder.js

```diff
@@ -5,6 +5,22 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num !== undefined) {
```

Comment:
The condition `if (num !== undefined)` is not entirely reliable. Consider using `arguments.length` to check if the function was called with arguments. This will help distinguish between a call with no arguments and a call with an argument explicitly set to `undefined`.